R-packages/evalcast/man-roxygen/signals-template.R

```diff
@@ -1,15 +1,43 @@
-#' @param signals Tibble specifying which variables from the COVIDcast API will be downloaded and passed along to `forecaster`.
-#'   Each row of `signals` represents a separate variable to be downloaded.  Available data
-#'   sources, signals, and geo resolutions are documented in the [COVIDcast signal
+#' @param signals Tibble with mandatory columns `data_source` and `signal` that 
```

Comment:
Suggest listing all possible columns at the start, then going into detail about them. Something like "mandatory columns `data_source` and `signal` and optional columns..."